src/Extensions/Logging/ILogEventSender.cs

```diff
@@ -30,6 +31,7 @@ public interface ILogEventSender
 		/// <param name="eventId">event Id</param>
 		/// <param name="threadId">Id of the thread</param>
 		/// <param name="message">Log message</param>
-		void LogMessage(Activity? activity, string category, LogLevel level, EventId eventId, int threadId, string message);
+		/// <param name="exception">The exception, if any, associated with the log.</param>
+		void LogMessage(Activity? activity, string category, LogLevel level, EventId eventId, int threadId, string message, Exception exception);
```

Comment:
I think `exception` should be marked as nullable her and in implementation, like `Exception? exception`, I'll also fix method in OmexLogger during formatter fix

Reply:
Ah, right, forgot about that you turned off nullable reference types